internal/jobcontainers/storage.go

```diff
@@ -75,3 +85,18 @@ func (c *JobContainer) setupRootfsBinding(root, target string) error {
 	}
 	return nil
 }
+
+// don't expose fileBindingSupportedOnce directly, since it can be reassigned.
+var fileBindingSupportedOnce = sync.OnceValue(func() bool {
+	// TODO: use windows.NewLazySystemDLL("bindfltapi.dll").Load() (or windows.LoadLibraryEx directly)
+
+	root := os.Getenv("SystemRoot")
+	if root == "" {
+		root = `C:\windows` // shouldn't really need this fall back, but ...
+	}
+	bindDLL := filepath.Join(root, `system32\bindfltapi.dll`)
+	st, err := os.Stat(bindDLL)
+	return err == nil && st.Mode().IsRegular()
+})
+
```

Comment:
Can we just do this in an `init` function and set a `FileBindingSupported` variable? 

Also, since this isn't a test code change, we should probably make it into a separate commit, in case we need to revert stuff.

Reply:
I can make a separate commit/PR, but my worry with exposing the variable directly is it can be (inadvertently) changed, which id like to prevent doing

Reply:
As long as the variable is package private it should be okay, I just find that approach easier to read & reason about. But I don't have any strong preference.

Reply:
Could you also make this a separate commit?

Reply:
done